api/v1/openapi.yaml

```diff
@@ -2159,3 +2180,43 @@ definitions:
       endpoint-id:
         description: The endpoint that made this lookup, or 0 for the agent itself.
         type: integer
+  IPCache:
+    description: IP cache definition and content
+    type: object
+    properties:
+      cache:
+        description: Contents of cache
+        type: array
+        items:
+          "$ref": "#/definitions/IPCacheEntry"
+  IPCacheEntry:
+    description: IP cache entry"
+    type: object
+    required:
+      - cidr
+      - identity
+    properties:
+      cidr:
+        description: Key of the entry in the form of a CIDR range
+        type: string
+      identity:
+        description: Numerical identity assigned to the IP
+        type: integer
+      hostIP:
+        description: IP address of the host
+        type: string
+      encryptKey:
+        description: The context ID for the encryption session
+        type: integer
+      k8sMetadata:
+        "$ref": "#/definitions/IPCacheEntryK8sMetadata"
+  IPCacheEntryK8sMetadata:
+    description: Kubernetes Metadata of an IP cache entry
```

Comment:
Does this need to be k8s-specific? Wondering if it could be a little more generic. Something like "source: k8s, name: xxx, namespace: yyy"